lib/vicadmin/validate.go

```diff
@@ -258,7 +259,7 @@ func (v *Validator) QueryDatastore(ctx context.Context, vch *config.VirtualConta
 	sort.Sort(dataStores)
 	if err != nil {
 		log.Errorf("Error while accessing datastore: %s", err)
-		return
+		return err
```

Comment:
Nothing, it's thrown out. It's just used to exit the function before we start trying to access `nil` members.